README.md

```diff
@@ -6,81 +6,74 @@ Note that over time, this job, and all the jobs called by it, will be migrated t
+Note that this approach will change, once a new system will be implemented, where we can more clearly specify dependencies between workflows, using special types of GitHub action.
+
+
+## Release procedure
 1. [Create new release issue to report status and collect any blocking issues](https://github.com/eclipse/che/issues/new?assignees=&labels=kind%2Frelease&template=release.md&title=Release+Che+7.FIXME)
-1. Update `VERSION` file in che-release repo's release branch, including enabling the PHASES (currently 1 - 7)
-1. Push commit to `release` branch (with `-f` force if needed)
-1. Wait until https://ci.centos.org/job/devtools-che-release-che-release/ completes (2-3 hrs)
-
-    TODO: add notification when build is done - email? slack? mattermost?
-
-1. NOTE: If the build should fail, check log for which step crashed (eg., due to [rate limiting](https://github.com/eclipse/che/issues/18292) or other errors), like E2E test failures. Sometimes simply re-running will help.
-
-1. To skip sections which have completed successfully, simply remove `PHASES` from the `VERSION` file and commit that change to the `release` branch to trigger the parts of the release that were not successful.
-
-## Phase 2 - manual steps
-
-### community operators
-
-TODO: [move this step into a GH action](https://github.com/eclipse/che-operator/pull/572). 
-TODO: set up the above action to fire when a PR like https://github.com/eclipse/che-operator/pull/571/files is merged, eg., 
-    when:
-        olm/eclipse-che-preview-openshift/deploy/olm-catalog/eclipse-che-preview-openshift/eclipse-che-preview-openshift.package.yaml and 
-        olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/eclipse-che-preview-kubernetes.package.yaml 
-    are updated
-
-This depends on the che-operator PRs being merged.
-
-1. Prepare for creation of community operator PRs via script in https://github.com/eclipse/che-operator (*ONLY after ALL PRs* for che-operator and chectl are merged):
-
-        ./olm/prepare-community-operators-update.sh
 
-1. Once created you'll see our PRs here:
-    * https://github.com/operator-framework/community-operators/pulls?q=%22Update+eclipse-che+operator%22+is%3Aopen
+2. To start a release, use the [Release - Orchestrate Overall Release Phases] workflow to begin with releasing . In the input, you have to provide the version of Che, DevWorkspace controller, and phases. 
 
-1. If tests fail or community operator PRs get stalled:
-            * Ping @j0zi (Jozef Breza) or @mavala (Martin Vala)
+2.1 If one of the workflows has crashed, inspect it. Apply fixes if needed, and restart it. You can restart individual workflow, or whole phase in orchestration job, whichever is simpler.
+2.2 Keep in mind, that sometimes you'll need to regenerate tags, or skip certain substeps in that job. Also ensure that correct code is in place, whether it is main or bugfix branch.
```

Comment:
link to issue about tag regeneration here with TODO? https://github.com/eclipse/che/issues/18879